spec/components/provider_interface/interview_and_course_summary_component_spec.rb

```diff
@@ -16,14 +16,24 @@
     expect(component).to include(interview.application_choice.course.provider.name)
   end
 
-  it 'displays the course name' do
-    expect(component).to include(interview.application_choice.course.name)
+  it 'displays the course name and code' do
+    expect(component).to include(interview.application_choice.course.name_and_code)
   end
 
   it 'displays interview location' do
     expect(component).to include(interview.location)
   end
 
+  it 'displays additional_details' do
```

Comment:
Can we move this and the next example in a common `additional_details` context so they are grouped together.